.gitignore

```diff
@@ -130,8 +132,4 @@ deps/uv/docs/src/guide/
 # ignore VS compiler output unhandled by V8's .gitignore
 deps/v8/gypfiles/Debug/
 deps/v8/gypfiles/Release/
-deps/v8/src/Debug/
-deps/v8/src/Release/
-deps/v8/src/inspector/Debug/
-deps/v8/src/inspector/Release/
```

Comment:
We don’t track these files, do we? Why are they being removed?

Reply:
Since the creation of `deps/v8/gypfiles` they are only created as subdirs there.
(In general they are only created in the location of `*.gyp` files (not even `*.gpyi`).